src/auth/digest/eDirectory/ldap_backend.cc

```diff
@@ -607,7 +607,7 @@ LDAPArguments(int argc, char **argv)
     if (!userbasedn || !((passattr != NULL) || (edir_universal_passwd && usersearchfilter && version == LDAP_VERSION3 && use_tls))) {
         fprintf(stderr, "Usage: " PROGRAM_NAME " -b basedn -f filter [options] ldap_server_name\n\n");
         fprintf(stderr, "\t-A password attribute(REQUIRED)\t\tUser attribute that contains the password\n");
-        fprintf(stderr, "\t-l password realm delimiter(REQUIRED)\tCharater(s) that devides the password attribute\n\t\t\t\t\t\tin realm and password tokens, default ':' realm:password\n");
+        fprintf(stderr, "\t-l password realm delimiter(REQUIRED)\tCharater(s) that divides the password attribute\n\t\t\t\t\t\tin realm and password tokens, default ':' realm:password\n");
```

Comment:
"Charater" also need a fix. Maybe manual later due to the \t issues.

Reply:
Yes. This is related to the \t issue.
Codespell is comparing the word "tcharater" against its default dictionary.
The \t and \n issue should be patched in codespell itself.
Maybe it is already being addressed?